sagemaker-fundamentals/create-training-job/container_kms/train.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+import boto3
+    
+import base64
+from cryptography.fernet import Fernet
+import os
+import json
+import pickle
```

Comment:
Don't use `pickle`

https://nedbatchelder.com/blog/202006/pickles_nine_flaws.html
https://lwn.net/Articles/595352/
https://intoli.com/blog/dangerous-pickles/

Reply:
Great content. I will highlight the disadvantage of pickle in review && discussion